src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -1,7 +1,6 @@
-<br>
-<ng-container>
-  <div class="card bg-primary">
-    <div class="card-body fill-plain">
+<div class="component">
```

Comment:
Please rename it to something else for e.g.`course-section`, `component` sounds too generic 😛

Reply:
Thanks haha I was trying to think of a better name but didn't think of one 🤷‍♀️